bootstrap-app/src/cmr/bootstrap/services/dispatch/core_async_dispatch.clj

```diff
@@ -0,0 +1,143 @@
+(ns cmr.bootstrap.services.dispatch.core-async-dispatch
```

Comment:
Some examples from the wild (done differently, but with similar code organization with regard to `impl`):
 * https://github.com/jonase/eastwood/tree/master/src/eastwood/copieddeps/dep10/clojure/tools/reader
 * https://github.com/clojure/core.async/tree/master/src/main/clojure/clojure/core/async
 * https://github.com/omcljs/om/tree/master/src/main/om/next
 * https://github.com/reagent-project/reagent/tree/master/src/reagent
 * https://github.com/funcool/catacumba/tree/master/src/clojure/catacumba
 * https://github.com/clojang/zhang/tree/release/0.1.0/src/zhang/process